l1-contracts/test/validator-selection/setupSampleSeed.t.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2025 Aztec Labs.
+pragma solidity >=0.8.27;
+
+import {ValidatorSelectionTestBase, CheatDepositArgs} from "./ValidatorSelectionBase.sol";
+import {Epoch, Timestamp} from "@aztec/core/libraries/TimeLib.sol";
+import {Checkpoints} from "@oz/utils/structs/Checkpoints.sol";
+import {IValidatorSelection} from "@aztec/core/interfaces/IValidatorSelection.sol";
+import {TestConstants} from "../harnesses/TestConstants.sol";
+
+contract SetupSampleSeedTest is ValidatorSelectionTestBase {
+  function test_setupSampleSeed(uint16 _epochToTest) public setup(4) {
+    // Check that the epoch is not set
+    bound(_epochToTest, 2, type(uint16).max);
```

Comment:
The bound is not actually used, you it returns a value so we can run into just throwing in 0 and then we end up underflowing further down